src/app/mini-games/sprint/components/sprint-game-end/sprint-game-end.component.ts

```diff
@@ -0,0 +1,36 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
+import { Answers } from 'src/app/core/models/ISprintGame';
+import { ITrainedWord } from 'src/app/core/models/ITrainedWord';
+
+@Component({
+  selector: 'app-sprint-game-end',
+  templateUrl: './sprint-game-end.component.html',
+  styleUrls: ['./sprint-game-end.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class SprintGameEndComponent {
+  @Input() trainedWords: ITrainedWord[] = [];
+  @Input() gamePoints = 0;
+  @Output() resetGame = new EventEmitter();
+  @Output() pronounceWord = new EventEmitter();
+
+  continuePlay() {
+    this.resetGame.emit();
+  }
+
+  countWrongWords() {
```

Comment:
Let's use type for a returning value too

Reply:
Added a returning value